remote-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+if [[ -z "${CARGO_TEST_CAMERA}" ]]; then
```

Comment:
What is the purpose of checking this?

Reply:
This overall would probably need more work. If a runner is defined, i.e. `target.aarch64-unknown-linux-gnu.runner` executables are always passed to the runner instead of executed directly. I was trying to create an escape hatch so that if someone doesn't have a camera, and hasn't set `CARGO_TEST_CAMERA`, it would still attempt to execute directly like normal. However, I hadn't really finished the script.